install_devconsole/consoledeveloper.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set +x
+OC_LOGIN_USERNAME=kubeadmin
+oc login -u ${OC_LOGIN_USERNAME} -p ${OC_LOGIN_PASSWORD}
+
+oc apply -f ./yamls/unmanage.yaml
+oc scale --replicas 0 deployment console-operator --namespace openshift-console-operator
+oc scale --replicas 0 deployment console --namespace openshift-console
+oc apply -f ./yamls/redeploy-console-operator.yaml
+#It takes time to get the pod in running state
+while [ "$(oc get pods --field-selector=status.phase=Running -n openshift-console-operator)" == "No resources found." ]
+do
+    sleep 1s
+done
+oc scale --replicas 1 deployment console --namespace openshift-console
```

Comment:
Could you add a loop which checks if the `deploymentconfig/console` has got updated with the new image?

Reply:
There is no deploymentconfig as such...  just the deployment and when we apply that redeploy file then the image is there. (The new-image is already there in the yaml..it is not being replaced)

Reply:
I wanted to imply `deployment/console` :) 

You are redeploying the `deployment/console-operator` not `deployment/console`. On re-deploying `deployment/console-operator`, the operator should update `deployment/console` which takes a while.

Reply:
After the redeployment the pod for this console-operator comes up. After this step the devconsole.sh script is called. So till the catalogsource and subscription is being applied that pod comes up and these two are kind of independent. so i thought of not putting it there. For pulling the latest image though I have a new commit which changes the imagePullPolicy to Always in the desired yaml and also does some changes in the consoledeveloper.sh

Reply:
@Avni-Sharma could you share the link to the workaround please? If there's an issue I would like to point people towards it every time they hit the bug.